spec/models/base_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+RSpec.describe Base do
+  describe "connection_class" do
+    before do
+      RequestStore.store[:request_id] = "some-request-id"
+    end
+
+    it "connection sends X-Request-Id header" do
+      stub = stub_request(:post, "http://localhost:3001/api/v3/bases")
+        .with(
+          body: "{\"data\":{\"type\":\"bases\",\"attributes\":{}}}",
+          headers: {
+            "Accept" => "application/vnd.api+json",
+            "Accept-Encoding" => "gzip,deflate",
+            "Content-Type" => "application/vnd.api+json",
+            "User-Agent" => "Faraday v0.17.3",
```

Comment:
Does this mean every update we need to bump this as well?

Reply:
At the moment yes, until someone reads the webmock docs and figures out how to only check the one header we need

Reply:
I've fixed it now, turned out it was way easier than i thought